document/core/text/types.rst

```diff
@@ -139,7 +154,16 @@ Memory Types
 .. math::
    \begin{array}{llclll@{\qquad\qquad}l}
    \production{memory type} & \Tmemtype &::=&
-     \X{lim}{:}\Tlimits &\Rightarrow& \X{lim} \\
+     \X{it}{:}\Tidxtype~~\X{lim}{:}\Tlimits &\Rightarrow& \X{it}~\X{lim} \\
+   \end{array}
+
+Abbreviations
+.............
+
+.. math::
+   \begin{array}{llclll@{\qquad\qquad}l}
+   \production{memory type} & \Tmemtype &::=&
+     \X{lim}{:}\Tlimits &\Rightarrow& \I32~\X{lim} \\
```

Comment:
```suggestion
   \production{index type} &
     \epsilon &\equiv& \I32 \\
```
This abbreviation should rather go to the section on index type (and then nothing needs to be said here or in the table type section).